packages/syft/src/syft/client/client.py

```diff
@@ -323,6 +329,213 @@ def get_client_type(self) -> type[SyftClient]:
             return SyftError(message=f"Unknown node type {metadata.node_type}")
 
 
+@serializable(
+    attrs=["proxy_target_uid", "dht_key", "vld_forward_proxy", "vld_reverse_proxy"]
+)
+class VeilidConnection(NodeConnection):
+    __canonical_name__ = "VeilidConnection"
+    __version__ = SYFT_OBJECT_VERSION_1
+
+    vld_forward_proxy: GridURL = GridURL.from_url(VEILID_SERVICE_URL)
```

Comment:
Can we use the `Field` class from pydantic.

from pydantic import Field

```python
vld_forward_proxy: GridURL = Field(default_factory=GridURL.from_url(VEILID_SERVICE_URL))
vld_reverse_proxy: GridURL = Field(default_factory=GridURL.from_url(VEILID_SYFT_PROXY_URL))
```

Reply:
Resolved by: [3c16d60](https://github.com/OpenMined/PySyft/pull/8526/commits/3c16d60e3956d1a4dacc54fcdee566ef3a18ab08)